### PR TITLE
Add LLVMPassManagerBuilderRef Extensions

### PR DESCRIPTION
**Pull request by talyian**
Add some missing methods on LLVMPassManagerBuilderRef

### CODE REVIEW

**Comment by EgorBo** [comment]
btw, does LlvmSharp support the new Pass manager?

**Comment by tannergooding** [comment]
LLVMSharp just wraps all the APIs publicly exposed by the underlying LLVM 8.0.0 header files.

AFAIK, all the bindings are declared correctly so they should work if they are exposed.

File: sources/LLVMSharp/Extensions/LLVMPassManagerBuilderRef.cs [hunk 1]
```diff
@@ -13,5 +13,32 @@ public partial struct LLVMPassManagerBuilderRef : IEquatable<LLVMPassManagerBuil
         public bool Equals(LLVMPassManagerBuilderRef other) => Pointer == other.Pointer;
 
         public override int GetHashCode() => Pointer.GetHashCode();
+
+        public void Dispose() {
```
Comment (tannergooding) [inline sources/LLVMSharp/Extensions/LLVMPassManagerBuilderRef.cs]: nit: This repo uses allman style braces (new line before the brace) as per the .editorconfig file:
https://github.com/microsoft/LLVMSharp/blob/master/.editorconfig#L254

If you are using an IDE or editor with a compatible .editorconfig extension and a supported Roslyn plugin (VS has one built-in and VS Code has one you can install), then the document should be automatically formatted with the correct rules as you are working.

**Comment by tannergooding** [comment]
Overall LGTM.

Just a nit where the coding style deviates from the .editorconfig.

**Comment by tannergooding** [comment]
Thanks!

**Pull request merged**